app/controllers/steps/duty_controller.rb

```diff
@@ -21,5 +23,11 @@ def uk_options
     def xi_options
       @xi_options ||= DutyCalculator.new(filtered_commodity(source: 'xi')).options
     end
+
+    def title
+      t('page_titles.duty_calculation') if @duty_options.present?
+
+      t('page_titles.no_duty')
+    end
```

Comment:
does this method always return `t('page_titles.no_duty')`? 
I think you missed a "return"

This could go too:
```
def title
   @duty_options.present? ? t('page_titles.duty_calculation') : t('page_titles.no_duty')
end
```